### PR TITLE
:sparkles: Verify PR title

### PR DESCRIPTION
**Pull request by djzager**
This creates a verifyPR action that we can include in Konveyor project workflows to ensure Pull Requests are titled correctly.

Signed-off-by: David Zager <david.j.zager@gmail.com>

### CODE REVIEW

File: cmd/verify-pr/action.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: 'Verify PR'
+description: 'Verify PRs for the KubeBuilder project repos & similar'
```
Comment (shawn-hurley) [inline cmd/verify-pr/action.yml:2]: nit: s/KubeBuilder/konveyor

File: cmd/verify-pr/verify-pr.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+func main() {
+
+	ghContext, err := action.VarsFromEnv()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Parse the event
+	event, err := func() (github.PullRequestEvent, error) {
```
Comment (shawn-hurley) [inline cmd/verify-pr/verify-pr.go:24]: Is there a reason to do this in an anonymous function? you could add the defer inline right?

Comment (djzager) [reply cmd/verify-pr/verify-pr.go:24]: No. Yes.

Parts copied without enough :brain:

**Comment by shawn-hurley** [comment]
this is perfect we are on the same page!

**Pull request merged**